examples/air-purifier-app/air-purifier-common/include/filter-delegates.h

```diff
@@ -18,7 +18,7 @@
 #include <app-common/zap-generated/ids/Attributes.h>
 #include <app-common/zap-generated/ids/Clusters.h>
 #include <app/clusters/resource-monitoring-server/resource-monitoring-cluster-objects.h>
-#include <app/clusters/resource-monitoring-server/resource-monitoring-server.h>
```

Comment:
For backwards compatibility, we should try to keep the old interface (that just loads the new one)